diff --git a/lib/events/ui-request.d.ts b/lib/events/ui-request.d.ts
index 041673460ea3aa3403e5d7df71f9c81a985af15b..3f29832216d47b87fac1a689c6eb87311d34f2bf 100644
--- a/lib/events/ui-request.d.ts
+++ b/lib/events/ui-request.d.ts
@@ -1,4 +1,3 @@
-import type { EventTypeDeviceSelected } from '@trezor/connect-analytics';
 import type { PROTO } from '../constants';
 import type { TransportDisableWebUSB, TransportRequestWebUSBDevice } from './transport';
 import type { Device, CoinInfo, BitcoinNetworkInfo } from '../types';
@@ -142,7 +141,7 @@ export interface UiRequestConfirmation {
             className: string;
             label: string;
         };
-        analytics?: EventTypeDeviceSelected;
+        analytics?: any;
     };
 }
 export interface UiRequestSelectDevice {
